### DISCUSSION TITLE
Meeting Notes for GSoC (6/18/2024)

### DISCUSSION THREAD

**Discussion by JackyZzZz** [theochem/Selector#215] [General]
# Meeting Notes

**Date:** 6/18/2024
**Participants:** @FanwangM   @JackyZzZz

## Meeting Overview

1. **Progress Update**
   - Deployment pipeline is mostly completed at this stage. Our application is live on [DockerHub](https://hub.docker.com/repository/docker/jackyzzz076/selector-deployment/general) and [HuggingFace](https://huggingface.co/spaces/JackyZzZzZ/selector) through a CI/CD pipeline with the support of Github Actions workflow.
   - Raised the first PR for code review, will merge it once everything is resolved. https://github.com/theochem/Selector/pull/212

## Next Steps

**To-Do Before Next Meeting:**
- Finish resolving any changes needed for the PR.
- Continue working on the interface for other algorithms.

**Comment by FanwangM** [comment]
I have put some comments for the review. 
Nice job!
